kola/harness.go

```diff
@@ -24,18 +24,32 @@ import (
 	"github.com/coreos/mantle/platform"
 )
 
-// tests should register by adding init() functions to this package
-var Tests []Test
-
-type Test struct {
-	Run         func(platform.Cluster) error
-	Name        string //Should be uppercase and unique
-	CloudConfig string
-	ClusterSize int
-	Platforms   []string // whitelist of platforms to run test against -- defaults to all
+// Defines a group of tests to run on a cluster instance. These tests are
+// repeated on a new cluster for each platform. Cluster test functions are run
+// first using the platform.Cluster interface. Then the series of native code
+// tests are run on each member of the cluster.
+type TestGroup struct {
```

Comment:
Could you split adding TestGroup and adding NativeTests to it into two commits please?